app/views/categories/show.html.erb

```diff
@@ -11,16 +11,18 @@
 <% end %>
 
 <% post_count = @posts.count %>
-<div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
-  <div>
+<div class="has-color-tertiary-500 category-meta">
+  <div title="<%= post_count %> posts">
     <%= short_number_to_human post_count, precision: 1, significant: false %>
-    <%= 'post'.pluralize(post_count) %> &middot;
-
+    <%= 'post'.pluralize(post_count) %> 
+  </div>
+  <div>
     <% if current_user&.is_admin %>
       <%= link_to 'Edit Category', edit_category_path(@category) %>
       &middot;
-    <% end %>
-
+      <% end %>
+  </div>
+  <div title="Get email notifications for new top-level posts">
```

Comment:
I like the addition of title text here, but using a div is going to cause the same problem of defaulting to being on its own line. Instead the title can be added directly to the `<% link_to` element, and the div can be removed.

As this bit is Ruby rather than HTML, it will need to be `title:` rather than `title=`, and the style seems to be to use single quotes rather than double quotes for the Ruby sections. So it will end up looking something like this:

```ruby
    <%= link_to 'Subscribe',
                new_subscription_path(type: 'category', qualifier: @category.id, return_to: request.path),
                class: 'button is-outlined',
                title: 'Get email notifications for new top-level posts' %>
```